sonar-plugin-api/src/main/java/org/sonar/api/server/rule/RulesDefinition.java

```diff
@@ -563,6 +571,8 @@ abstract class Rule {
      */
     public abstract List<RuleDescriptionSection> ruleDescriptionSections();
 
+    public abstract Set<String> genericConceptKeys();
```

Comment:
missing javadoc

Reply:
are you expecting more than:
`   /**
     * @since 9.8
     */
`

Reply:
my comment was mostly about the `since`, however a small description would probably be helpdul